src/query/service/src/servers/mysql/mysql_handler.rs

```diff
@@ -103,29 +104,38 @@ impl MySQLHandler {
     }
 
     fn accept_socket(
-        sessions: Arc<SessionManager>,
+        session_manager: Arc<SessionManager>,
         executor: Arc<Runtime>,
         socket: TcpStream,
         keepalive: TcpKeepalive,
         tls: Option<Arc<ServerConfig>>,
     ) {
-        executor.spawn(async move {
-            match sessions.create_session(SessionType::MySQL).await {
+        executor.spawn(GLOBAL_TASK, async move {
```

Comment:
No need GLOBAL_TASK, changed at: https://github.com/datafuselabs/databend/pull/15495)

Reply:
got it